File: src/main/java/top/dsbbs2/bukkitcord/api/IPlayer.java
package top.dsbbs2.bukkitcord.api;

import java.net.InetSocketAddress;
import java.util.*;

public interface IPlayer extends ICommandSender {
    UUID getUniqueId();
    String getServerName();
    ArrayList<IPlayer> getServerOnlinePlayers();
    ArrayList<String> getServerOnlinePlayerNames();
    void connect(String serverName);
    String getPlayerPrefix();
    String getPlayerSuffix();
    void chat(String mess);
    void sendBungeeCordPlayerMessage(IPlayer p, String message);
    int getPing();
    boolean pingIsInRange(int var1,int var2);
    InetSocketAddress getAddress();
}

File: src/main/java/top/dsbbs2/bukkitcord/api/IPlayer.kt
package top.dsbbs2.bukkitcord.api

import java.net.InetSocketAddress
import java.util.*

interface IPlayer : ICommandSender {
    val uniqueId: UUID?
    val serverName: String?
    val serverOnlinePlayers: ArrayList<IPlayer?>?
    val serverOnlinePlayerNames: ArrayList<String?>?
    fun connect(serverName: String?)
    val playerPrefix: String?
    val playerSuffix: String?
    fun chat(mess: String?)
    fun sendBungeeCordPlayerMessage(p: IPlayer?, message: String?)
    fun getPing():Int
    fun pingIsInRange(var1:Int?,var2:Int?):Boolean
    fun getAddress(): InetSocketAddress
}